src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -620,6 +620,18 @@ public static void setSingleSchedulerHandler(@Nullable Function<? super Schedule
         onSingleHandler = handler;
     }
 
+    /**
+     * Unwraps internal scheduler's task.
+     * @param task the internal task
+     * @return the unwrapped runnable or task
+     */
```

Comment:
Please add `@since 2.1.7 - experimental` and `@Experimental` annotation to the method.

Reply:
Alos please add `@Nullable`  and `@CheckReturnValue` to the return type, `@NonNull` to the parameter.

Reply:
ok